interfaces/polkit/backend_test.go

```diff
@@ -80,6 +81,41 @@ func (s *backendSuite) TestInstallingSnapWritesPolicyFiles(c *C) {
 	}
 }
 
+func (s *backendSuite) TestInstallingSnapWritesRuleFiles(c *C) {
```

Comment:
Can a test be added which adds a rule with an invalid namesuffix to ensure it doesn't get written out?